spotlight/factorization/explicit.py

```diff
@@ -80,7 +82,7 @@ def __init__(self,
         self._l2 = l2
         self._use_cuda = use_cuda
         self._sparse = sparse
-        self._optimizer = None
+        self._optimizer = optimizer
```

Comment:
Yes, I think every call to `fit` right now will basically re-set the model. Something to look into, we probably want an ability to continue.